### ISSUE TITLE
Error when uploading MOT annotations

### ISSUE THREAD

**Issue by yasakova-anastasia** [opencv/cvat#4835]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour
Upload MOT annotations normally.

### Current Behaviour
I have MOT annotations saved in `.zip` format with the following structure

```
$unzip -l tracking_results.zip 
Archive:  tracking_results.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
    24059  2022-08-13 13:01   gt.txt
        3  2022-08-13 13:01   labels.txt
---------                     -------
    24062                     2 files
```

Here are the first few lines of the `gt.txt` file

```
0,0,0.7899999618530273,304.20000553131104,98.29000234603882,95.31000256538391,1,1,1.0
1,0,3.7700001150369644,301.1899948120117,98.28000068664551,95.31000256538391,1,1,1.0
2,0,6.7500002682209015,298.1800079345703,98.28000068664551,95.31000256538391,1,1,1.0
3,0,9.719999879598618,295.16000747680664,98.29000234603882,95.31999826431274,1,1,1.0
4,0,12.700000405311584,292.1499967575073,98.28000068664551,95.31000256538391,1,1,1.0
5,0,15.680000185966492,289.1400098800659,98.28000068664551,95.31000256538391,1,1,1.0
6,0,18.649999797344208,286.1299991607666,98.29000234603882,95.31000256538391,1,1,1.0
7,0,21.629999577999115,283.1199884414673,98.28000068664551,95.31000256538391,1,1,1.0
8,0,24.609999358654022,280.1000118255615,98.28000068664551,95.31999826431274,1,1,1.0
9,0,27.57999897003174,277.0900011062622,98.29000234603882,95.31000256538391,1,1,1.0
```
and the `labels.txt` file is simply

```
ok
```

However, when I try to upload these annotations using `Upload annotations` -> `MOT 1.1` (on [cvat.org](cvat.org) I get an error saying

```
Error: Request failed with status code 500. "datumaro.components.errors.DatasetNotFoundError: Failed to find dataset at '/tmp/tmpcntvb5ty'\n".
```
What am I doing wrong? I think the format follows the one listed [in the documentation](https://openvinotoolkit.github.io/cvat/docs/manual/advanced/formats/format-mot/).

[Original issue](https://github.com/cvat-ai/cvat/issues/102) by @tadejsv

**Comment by kirill-sizov** [comment]
Issue successfully reproduced by me

The problem is that Datumaro searches annotation files in `gt` directory, but if annotation files stores in root of archive it's throw an error. 

Another problem here is that provided annotation file uses incorrect frame numbering, according [MOT specification](https://motchallenge.net/instructions/) frame numbers should start from `1`

**Comment by zhiltsov-max** [comment]
@kirill-sizov , can we close the issue? Do you expect any fixes for this?

**Comment by kirill-sizov** [comment]
@zhiltsov-max Yes, I think we should support this use case, when annotations files are stored in the root of archive without `gt` sub-directory for MOT format

**Comment by niqbal996** [comment]
Hi @kirill-sizov, 
I am facing a similar issue. I have created a a labelling job with ANY kind of labels. I am uploading the labels in COCO 1.1 format as a zip file. I have checked the label file locally with the official COCO API and the labels show up as expected. But CVAT throws an error saying: 
```bash
cvat-app.tsx:295 Error: Request failed with status code 500. "Exception: Failed to find 'coco' dataset at '/tmp/tmpc61gumsx'\n"..
``` 
I do not think there is any fix posted here in this thread. I am using CVAT release version 2.2. Your help will be appreciated. 
Thanks a lot.

**Comment by WeiChihChern** [comment]
Got same issue as @niqbal996.

**Comment by zhiltsov-max** [comment]
@niqbal996, if possible, please share the annotations you're uploading, so we could reproduce the problem. We don't need the images, but info about the file structure in the archive could be helpful (such as file tree).

**Comment by niqbal996** [comment]
Hi @zhiltsov-max , 
Here is the annotation file I am using 
[nirannotations.zip](https://github.com/opencv/cvat/files/10094331/nirannotations.zip)
Thanks for looking into this.

**Comment by rosebbb** [comment]
> Hi @zhiltsov-max , Here is the annotation file I am using [nirannotations.zip](https://github.com/opencv/cvat/files/10094331/nirannotations.zip) Thanks for looking into this.

Did you solve this? I have exactly the same problem with COCO format annotation file

**Comment by zhiltsov-max** [comment]
Hi, there is no fix yet. Please make sure your dataset file layout corresponds to the one described [here](https://opencv.github.io/cvat/docs/manual/advanced/formats/).

**Comment by ixez** [comment]
It happened to me, but I figured out that I need to put `gt.txt` and `labels.txt` in a folder of `gt` inside the zip file.

**Comment by zhiltsov-max** [comment]
Related: https://github.com/opencv/cvat/issues/7052

**Comment by himanshu-sharmav** [comment]
Hi @zhiltsov-max I would love to work on this issue

**Comment by himanshu-sharmav** [comment]
Thanks @zhiltsov-max , can I have some clarity on what we sought to achieve?

**Comment by zhiltsov-max** [comment]
@himanshu-sharmav, CVAT currently supports only this input file layout for MOT-like annotations: https://docs.cvat.ai/docs/manual/advanced/formats/format-mot/#mot-import. The simplified flattened layout without the `gt/` dir could also be supported as an extension for annotation uploads.
